test/linalg/symmetric.jl

```diff
@@ -42,206 +41,229 @@ end
         x = eltya == Int ? rand(1:7, n) : convert(Vector{eltya}, eltya <: Complex ? complex.(x, zeros(n)) : x)
         y = eltya == Int ? rand(1:7, n) : convert(Vector{eltya}, eltya <: Complex ? complex.(y, zeros(n)) : y)
         b = eltya == Int ? rand(1:7, n, n) : convert(Matrix{eltya}, eltya <: Complex ? complex.(b, zeros(n,n)) : b)
+        @testset "basic ops" begin
+            @testset "constructor" begin
+                @test Symmetric(Symmetric(asym, :U))     === Symmetric(asym, :U)
+                @test Hermitian(Hermitian(asym, :U))     === Hermitian(asym, :U)
```

Comment:
`aherm` here and on all other places for `Hermitian` in the `"constructor"` and `"similar"` testsets.